### PR TITLE
fix zero size element vector deserialize security issue

### PR DESCRIPTION
**Pull request by ailisp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Why do we need yet another serialization format? Borsh is the first serializer t
    * Consistent means there is a bijective mapping between objects and their binary representations. There is no two binary representations that deserialize
    into the same object. This is extremely useful for applications that use binary representation to compute hash;
    * Borsh comes with a full specification that can be used for implementations in other languages;
-* Safe. Borsh implementations use safe coding practices. In Rust, Borsh uses only safe code;
```
Comment (nearmax) [inline README.md]: Also the website, please.

Comment (nearmax) [reply README.md]: See `docs`.

Comment (ailisp) [reply README.md]: Thanks. just updated, is website auto deployed?

Comment (ailisp) [reply README.md]: sorry seems it's auto reformat on save by editor, i'll change it back now

Comment (ailisp) [reply README.md]: fixed

**Pull request merged**